src/interface/mod.rs

```diff
@@ -48,6 +48,31 @@ pub trait Interner<K = Spur>: Reader<K> + Resolver<K> {
     fn try_get_or_intern_static(&mut self, val: &'static str) -> LassoResult<K>;
 }
 
+impl<T, K> Interner<K> for &mut T
+where
+    T: Interner<K>,
+{
+    #[cfg_attr(feature = "inline-more", inline)]
```

Comment:
Honestly you can probably just make these `#[inline]` directly since they're just dispatch calls